src/map/status.c

```diff
@@ -135,21 +135,8 @@ static void initChangeTables(void)
 	memset(status->dbs->DisplayType, 0, sizeof(status->dbs->DisplayType));
 
 	// Storing the target job rather than simply SC_SOULLINK simplifies code later on.
```

Comment:
may you move this comment to the new function? since it is explaining why we are doing something weird (casting a `MAPID` to `sc_type`), I think it makes more sense to keep it where the thing is actually being done